src/stories/BalancesCalendar.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import {storiesOf} from '@storybook/react';
+import moment from 'moment';
+import React from 'react';
+import GlobalWrapper from '../helpers/GlobalWrapper';
+
+import BalancesCalendarEntry from '../components/atoms/BalancesCalendarEntry';
+import BalancesCalendar from '../components/molecules/BalancesCalendar';
+import { IBalanceEntry } from '../declarations/balanceEntries';
+
+const entry: IBalanceEntry = {
+  date: '2019-06-05',
+  expense: 200000,
+  income: 200000,
+  liquidity: 18000000,
+};
+
+const entries: Array<IBalanceEntry> = [
```

Comment:
```suggestion
const entries: Array<IBalanceEntry> = (new Array(13).fill(0)).map((_e, i) => ({
  date: `2019-06-${String(i + 1).padStart(2, '0')}`,
  expense: 200000,
  income: 200000,
  liquidity: 18000000,
}));
```

Might be a bit hacky, but a lot terser.

Reply:
During development the array was always manipulated for different scenarios both in terms of dates and different combinations of expense, income and liquidity, where i found the hardcoded array approach much simpler. As of now I see no use for that anymore and think your suggestion fits better. Changed your variable '_e' to 'e' to fix tslint complaints.
Fixed.

Reply:
Aha, makes sense.

As for the `_e`, I'm just used to a leading underscore meaning that the variable is not used, but apparently tslint doesn't agree 🤔